.github/workflows/release.yaml

```diff
@@ -130,21 +127,8 @@ jobs:
       with:
         fetch-depth: 1
 
-    - uses: actions/setup-go@v4
-      with:
-        go-version: 1.18.x
-
-    - name: Run ensure
-      run: |
-        make deps
-        make ensure
-
-    - name: Build and package darwin arm64 binary
-      run: |
-        make build-package-darwin-arm64
-
     - name: Upload MacOS arm64 binary & checksum
       uses: softprops/action-gh-release@v1
       with:
         files: |
-          bin/**
+          target/bin/darwin/arm64/**
```

Comment:
tää ei toimi näin, eli kun nää jokanen "publish" steppi on oma jobinsa -> ne ajautuu omalla runnerilla -> tuota buildattua target kansiota ei ole olemassa. Sen sijaan jos haluat käyttää tota subsettia niin nämä extra publishit pitäs poistaa ja laittaa kaikki samaan jobiin missä on toi subsetin buildaus

Reply:
Joo, tehäänpä niin

Reply:
Noniin, nyt ne on samassa jobissa ja nyt tehään tuo checksum mikä jäi aiemmasta. Noi on nyt kaikki nimeltään `resource-advisor`, tuleeko jotain konfliktia uploadatessa? Pitäskö ne nimetä samaan tapaan ku aiemmin?

Reply:
tulee konflikti, muuten näyttää ihan hyvältä

Reply:
Nyt olis